cmake/Modules/FindRTICivetweb.cmake

```diff
@@ -0,0 +1,139 @@
+# (c) 2023 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the software solely for use with RTI Connext DDS.  Licensee may
+# redistribute copies of the software provided that all such copies are
+# subject to this license. The software is provided "as is", with no warranty
+# of any type, including any warranty for fitness for any purpose. RTI is
+# under no obligation to maintain or support the software.  RTI shall not be
+# liable for any incidental or consequential damages arising out of the use or
+# inability to use the software.
+
+#[[.rst:
+.. _find_rticivetweb:
+
+FindRTIOCivetweb
+--------------
+
+Find the Civetweb libraries.
+If no RTICivetweb_ROOT is provided it will try to find the libraries in the
+Connext installation.
+
+The list of paths to search the libraries are:
+
+- ``RTICivetweb_ROOT``
+- Environment variable ``RTICivetweb_ROOT``
+- ``${CONNEXTDDS_DIR}/third_party/civetweb-<version>/${CONNEXTDDS_ARCH}/<build_mode>``
+
+Output variables related to libraries:
```

Comment:
The output variables are related to the following libraries:

(at least, I think that's what you mean)

Reply:
What about:
> The output variables related to the Civetweb libraries are:
So I'm trying to say "These are the variables that hold the library paths and the imported target variables"?